cirq/google/arg_func_langs_test.py

```diff
@@ -0,0 +1,155 @@
+# Copyright 2019 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import pytest
+import sympy
+
+from google.protobuf import json_format
+
+import cirq
+from cirq.google.arg_func_langs import (
+    _arg_from_proto,
+    _arg_to_proto,
+    ARG_LIKE,
+    LANGUAGE_ORDER,
+)
+from cirq.api.google import v2
+
+
+@pytest.mark.parametrize('min_lang,value,proto', [
```

Comment:
I've seen where combining test data into a common place goes, and it's not good. The common data grows and grows in its complications as more tests use it, everyone adds on their own little parameter to make their one test work, and ultimately individual tests become totally incomprehensible because you don't know which properties matter and which don't for that test.

Tests details should be kept very local, even if that results in code duplication. There are many exceptions to this, but the underlying idea of "tolerate more duplication in tests" is a good rule of thumb.